python/src/aiconfig/editor/client/src/themes/GradioTheme.ts

```diff
@@ -149,133 +138,141 @@ export const GRADIO_THEME: MantineThemeOverride = {
-          backgroundColor: theme.colorScheme === "light" ? "white" : "#384152",
+          border: "1px solid #FDD7AD",
+          background: "linear-gradient(180deg, #FEE1C0 0%, #FCC792 100%)",
+          boxShadow: "0px 1px 4px 0px rgba(0, 0, 0, 0.05)",
+          margin: "4px",
+          marginTop: "30px",
+          padding: "0.625rem !important",
+          height: "auto",
+        },
+
+        ".actionTabsPanel": {
+          width: "400px",
         },
-        textarea: {
-          border: `1px solid ${inputBorderColor} !important`,
-          boxShadow: "0px 1px 4px 0px rgba(0, 0, 0, 0.05) inset",
+
+        ".parametersContainer": {
+          maxWidth: "1250px",
+          maxHeight: "-webkit-fill-available",
+          margin: "16px auto 16px 36px",
+          padding: "0",
+          backgroundColor:
+            theme.colorScheme === "light" ? "#F9FAFB" : "#1f2938",
           borderRadius: "8px",
-          backgroundColor: theme.colorScheme === "light" ? "white" : "#384152",
-          ":focus": {
-            outline: "solid 1px #E85921 !important",
-            outlineOffset: "-1px",
+          border: `1px solid ${inputBorderColor}`,
+          button: {
+            ":hover": {
+              backgroundColor:
+                theme.colorScheme === "light" ? "#F0F1F1" : "transparent",
+            },
           },
-        },
-        ".addParameterButton": {
-          position: "sticky",
-          left: "0",
-          bottom: "0",
-          margin: "16px 0 0 0",
-          background: "linear-gradient(180deg, #FEE1C0 0%, #FCC792 100%)",
-          path: {
-            color: "#E85921",
+          input: {
+            border: `1px solid ${inputBorderColor} !important`,
+            boxShadow: "0px 1px 4px 0px rgba(0, 0, 0, 0.05) inset",
+            borderRadius: "8px",
+            backgroundColor:
+              theme.colorScheme === "light" ? "white" : "#384152",
+          },
+          textarea: {
+            border: `1px solid ${inputBorderColor} !important`,
+            boxShadow: "0px 1px 4px 0px rgba(0, 0, 0, 0.05) inset",
+            borderRadius: "8px",
+            backgroundColor:
+              theme.colorScheme === "light" ? "white" : "#384152",
+            ":focus": {
+              outline: "solid 1px #E85921 !important",
+              outlineOffset: "-1px",
+            },
+          },
+          ".addParameterButton": {
+            position: "sticky",
+            left: "0",
+            bottom: "0",
+            margin: "16px 0 0 0",
+            background:
+              "linear-gradient(to bottom right, #ffedd5, #fdba74 100%)",
+            border: "1px solid #fed7aa",
+            boxShadow: "0px 1px 4px 0px rgba(0, 0, 0, 0.05)",
+            color: "#ea580c !important",
+            path: {
+              color: "#E85921",
+            },
+            ":hover": {
+              background: "linear-gradient(to bottom right, #ffedd5, #fed7aa)",
+            },
```

Comment:
In general, how do you know how to add this stuff? Do you find it from somewhere, or do you just play around and figure it out? If you find it from somewhere, how do you extra the values on webpages etc?

Reply:
Most of this was from Zak's existing styles. But generally you can just go and 'inspect element' in existing gradio spaces to see what the styles are. Technically gradio uses style variables for a lot of things (e.g. `var(input-background-color-fill)` but those variables aren't declared for us locally so just using the explicit value of those variables where applicable